pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -356,6 +500,12 @@ func TestUserSignupWithNoApprovalPolicyTreatedAsManualApproved(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "foo",
 			Namespace: operatorNamespace,
+			Annotations: map[string]string{
```

Comment:
This object meta is duplicated in many places. Could we replace it by a method?
```
ObjectMeta: newObjectMeta("foo"),
```
or
```
ObjectMeta: newObjectMeta(""),
```
```
func newObjectMeta(name string) metav1.ObjectMeta {
    if name == "" {
        name = uuid.NewV4().String()
    }
    meta := metav1.ObjectMeta {
        Name:      name,
			Namespace: operatorNamespace,
			Annotations: map[string]string{
				toolchainv1alpha1.UserSignupUserEmailAnnotationKey: "foo@redhat.com",
			},
			Labels: map[string]string{
				toolchainv1alpha1.UserSignupUserEmailHashLabelKey: "fd2addbd8d82f0d2dc088fa122377eaa",
			},
	}
}
```

Reply:
I've implemented this where it makes sense.